### PR TITLE
refactor: add new module for event intake logic

### PR DESCRIPTION
**Pull request by IvanKavaldzhiev**
**Description**:
This PR adds a new module for consensus event intake logic, where all related logic is going to be extracted in.

**Related issue(s)**: #15643 

Fixes #

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

File: platform-sdk/consensus-event-intake/src/main/java/com.consensus.event.intake/EventIntakeService.java [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.consensus.event.intake;
+
+/**
+ * Mock event intake service.
+ */
+public interface EventIntakeService {}
```
Comment (litt3) [inline platform-sdk/consensus-event-intake/src/main/java/com.consensus.event.intake/EventIntakeService.java:22]: Do we need the word `Service` in this name? Why not just `EventIntake`?

**Comment by litt3** [comment]
@IvanKavaldzhiev Just a reminder - the correct branch naming format is a 5 digit issue number (no leading `0`).

**Comment by IvanKavaldzhiev** [comment]
Closing due to wrong branching name

**Comment by IvanKavaldzhiev** [comment]
[Link](https://github.com/hashgraph/hedera-services/pull/16058) to the new PR

**Pull request closed without merging**